_pkgdown.yml

```diff
@@ -2,3 +2,16 @@ url: https://gdrplatform.github.io/gDRtestData/
 template:
   bootstrap: 5
 
+reference:
+- title: Generate data
```

Comment:
Could you add short description of these function groups?

Reply:
I think after the latest changes: 
 - titles are telling and self-explanatory
 - users can get detailed info about all three categories by looking into the 'Get started' section.